# PWD: packages/apps/Messaging

diff --git a/Android.mk b/Android.mk
index bd4063c..618a3ff 100644
--- a/Android.mk
+++ b/Android.mk
@@ -74,17 +74,6 @@ LOCAL_MODULE_PATH := $(TARGET_OUT_PRODUCT_APPS)
 
 LOCAL_COMPATIBILITY_SUITE := general-tests
 
-LOCAL_REQUIRED_MODULES += com.android.messaging_whitelist.xml
-
 include $(BUILD_PACKAGE)
 
-include $(CLEAR_VARS)
-LOCAL_MODULE := com.android.messaging_whitelist.xml
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_PATH := $(TARGET_OUT_PRODUCT_ETC)/sysconfig
-LOCAL_PRODUCT_MODULE := true
-LOCAL_SRC_FILES := $(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
 include $(call all-makefiles-under, $(LOCAL_PATH))
diff --git a/com.android.messaging_whitelist.xml b/com.android.messaging_whitelist.xml
deleted file mode 100644
index 6bc77a8..0000000
--- a/com.android.messaging_whitelist.xml
+++ /dev/null
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-     Copyright (C) 2019-2020 The LineageOS Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-<config>
-    <allow-in-power-save package="com.android.messaging" />
-</config>
diff --git a/src/com/android/messaging/datamodel/action/ActionServiceImpl.java b/src/com/android/messaging/datamodel/action/ActionServiceImpl.java
index d78bdc0..172e792 100644
--- a/src/com/android/messaging/datamodel/action/ActionServiceImpl.java
+++ b/src/com/android/messaging/datamodel/action/ActionServiceImpl.java
@@ -171,7 +171,7 @@ public class ActionServiceImpl extends JobIntentService {
                     (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
 
             if (delayMs < Long.MAX_VALUE) {
-                mgr.set(AlarmManager.ELAPSED_REALTIME_WAKEUP,
+                mgr.setAndAllowWhileIdle(AlarmManager.ELAPSED_REALTIME_WAKEUP,
                         SystemClock.elapsedRealtime() + delayMs, pendingIntent);
             } else {
                 mgr.cancel(pendingIntent);
