# PWD: frameworks/base
diff --git a/services/core/java/com/android/server/power/Notifier.java b/services/core/java/com/android/server/power/Notifier.java
index 5a6229c..fcbe34a 100644
--- a/services/core/java/com/android/server/power/Notifier.java
+++ b/services/core/java/com/android/server/power/Notifier.java
@@ -99,7 +99,7 @@ public class Notifier {
     private static final int MSG_SCREEN_POLICY = 7;
     private static final int MSG_WIRED_CHARGING_DISCONNECTED = 8;
     private static final int MSG_WIRELESS_CHARGING_INTERRUPTED = 9;
-
+    /*
     private static final long[] CHARGING_VIBRATION_TIME = {
             40, 40, 40, 40, 40, 40, 40, 40, 40, // ramp-up sampling rate = 40ms
             40, 40, 40, 40, 40, 40, 40 // ramp-down sampling rate = 40ms
@@ -107,10 +107,9 @@ public class Notifier {
     private static final int[] CHARGING_VIBRATION_AMPLITUDE = {
             1, 4, 11, 25, 44, 67, 91, 114, 123, // ramp-up amplitude (from 0 to 50%)
             103, 79, 55, 34, 17, 7, 2 // ramp-up amplitude
-    };
+    };*/
     private static final VibrationEffect CHARGING_VIBRATION_EFFECT =
-            VibrationEffect.createWaveform(CHARGING_VIBRATION_TIME, CHARGING_VIBRATION_AMPLITUDE,
-                    -1);
+            VibrationEffect.createOneShot(200, VibrationEffect.DEFAULT_AMPLITUDE);
     private static final AudioAttributes VIBRATION_ATTRIBUTES = new AudioAttributes.Builder()
             .setContentType(AudioAttributes.CONTENT_TYPE_SONIFICATION)
             .build();
