# PWD: system/libbase

diff --git a/logging.cpp b/logging.cpp
index 5bd21da66..4c25a8daf 100644
--- a/logging.cpp
+++ b/logging.cpp
@@ -476,11 +476,6 @@ LogMessage::LogMessage(const char* file, unsigned int line, LogSeverity severity
     : data_(new LogMessageData(file, line, severity, tag, error)) {}
 
 LogMessage::~LogMessage() {
-  // Check severity again. This is duplicate work wrt/ LOG macros, but not LOG_STREAM.
-  if (!WOULD_LOG(data_->GetSeverity())) {
-    return;
-  }
-
   // Finish constructing the message.
   if (data_->GetError() != -1) {
     data_->GetBuffer() << ": " << strerror(data_->GetError());
